Alpaca.Markets/Enums/SortDirection.cs

```diff
@@ -5,21 +5,21 @@
 namespace Alpaca.Markets
 {
     /// <summary>
-    /// Order statuses sorting direction for <see cref="RestClient.ListOrdersAsync"/> call from Alpaca REST API.
+    /// Supported sort directions in Alpaca REST API.
     /// </summary>
     [JsonConverter(typeof(StringEnumConverter))]
-    public enum OrderListSorting
+    public enum SortDirection
```

Comment:
Type renaming is a breaking change and we have to change at least minor (ideally - major) version number of SDK package for signaling clients about this. I'm OK in general with this renaming but if we want to support a new endpoint in previous versions of SDK we have to keep `OrderListSorting`, mark it as `Obsolete` and remove it only in later releases. I suggest to keep it as is and I'll take care of these backward compatibility steps during a backporting work.

Reply:
Done.